torchrl/objectives/returns/advantages.py

```diff
@@ -20,9 +20,14 @@
 from torchrl.data.tensordict.tensordict import _TensorDict
 from torchrl.envs.utils import step_tensordict
 from torchrl.modules import TDModule
```

Comment:
TDModule(TensorDict) here actually may be confused as temporal difference module, as someone mentioned before I think in the discusstion

Reply:
Yeah, for sure.